### ISSUE TITLE
[Bug]: RBAC Generates playbooks for inactive ROLE credentials

### ISSUE THREAD

**Issue by fersy02** [apisec-inc/Support#29]
### Severity

Non-Blocker

### What happened?

Deactivating an RBAC authentication helps us in excluding roles. We honor the request and include the active authentications upon running an RBAC scan with automap enabled. However, once the map is ready, we do regenerate playbooks. It includes inactive credentials that were initially not part of the scan.

The Role Mapping contains the allowed permissions, and the disallowed is populated automatically. 

- [x] We shall ignore the inactive roles during the RBAC playbook generation. 
- [x] Updated RBAC playbooks shall ignore the execution of inactive roles.

**Comment by fersy02** [comment]
Any updates on this ticket?

**Comment by muqeetkhan20** [comment]
Already in prod, closing the ticket.

**Comment by muqeetkhan20** [comment]
@fersy02 we have inactivated one of the role (ROLE_PM) and we are not getting that role in playbook execution.
![image](https://user-images.githubusercontent.com/68276700/163981865-c749a3f1-399b-4124-ab06-a66ce1bef5f6.png)

Only two roles after scanning.
![image](https://user-images.githubusercontent.com/68276700/163982071-11643b43-deff-49bc-97fb-43549d0cabc8.png)

**Comment by fersy02** [comment]
Please attach the screenshot of the playbook to check the value in the Auth section.

**Comment by muqeetkhan20** [comment]
![image](https://user-images.githubusercontent.com/68276700/164185867-b78f045f-d922-403c-ac71-8c53d66535e8.png)

**Comment by fersy02** [comment]
As you see the Auth is empty. You need to regenerate playbooks and try this scenario.

**Comment by muqeetkhan20** [comment]
okay, this scenario is reproducible.
Spoke to @Mohammed-Luqman-Shareef - It should be available in next prod refresh.

**Comment by fersy02** [comment]
Please prioritize it as the client is pushing us to get this working at the earliest.

**Comment by Mohammed-Luqman-Shareef** [comment]
It is already in Prod.  
Once you activate/deactivate an RBAC regenerate the playbooks from the categories page by clicking "Save and rewrite playbooks"

**Comment by fersy02** [comment]
Both the use cases are working as desired. Informed Client. QA may close this ticket.
